Mac-App/Mac-App/Transformers/DetailsViewTransformer.swift

```diff
@@ -25,30 +40,38 @@ final class DetailsViewTransformer: StoreSubscriber {
         }
     }
 
-    private func mapAppStateToViewData(_ appState: AppState) -> DetailsViewTransformer.Status {
-        guard appState.dependencyGraphState.tree.count > 0 else {
+    private func mapAppStateToViewData(_ data: ProjectFactsViewData.Data) -> DetailsViewTransformer.Status {
```

Comment:
I just realised this echo `...ViewData.Data` I will rename this to follow something I'm already using like: `DependencyTreeView.Data` in a separate PR.

Reply:
Yeah sounds good 👍